diff --git a/pkg/v1/remote/descriptor_test.go b/pkg/v1/remote/descriptor_test.go
index c0e6fa25..3d554e63 100644
--- a/pkg/v1/remote/descriptor_test.go
+++ b/pkg/v1/remote/descriptor_test.go
@@ -175,7 +175,7 @@ func TestHeadSchema1(t *testing.T) {
 	}
 
 	if desc.Size != int64(len(response)) {
-		t.Errorf("Descriptor.Size = %q, expected %q", desc.Size, len(response))
+		t.Errorf("Descriptor.Size = %d, expected %d", desc.Size, len(response))
 	}
 }
 
